docs/Users_Guide/lulcncarhpc.rst

```diff
@@ -0,0 +1,184 @@
+
+These commands download three container images: the METplus software, the observational data, and the WRF simulation data. Processing the METplus software can take up to 15 minutes and the data containers can take several minutes each.
+
+Run METplus
+===========
+
+After the container images have been downloaded, you can run the METplus verification to compare the I-WRF LULC simulation results against observational data and generate statistical verification results and visualization plots. This process involves configuring the data bindings and executing the verification workflow for two meteorological variables.
+
+Configure Container Data Bindings
+---------------------------------
+
+METplus requires access to input data, configuration files, and output directories. Apptainer uses bind mounts to make local directories and container images available inside the running container. Set up the environment variables that define these data bindings.
+
+First, define the local directory paths for configuration and visualization scripts::
+
+    LOCAL_METPLUS_CONFIG_DIR=${IWRF_WORK_DIR}/i-wrf/use_cases/Land_Use_Land_Cover/METplus
+    LOCAL_PLOT_SCRIPT_DIR=${IWRF_WORK_DIR}/i-wrf/use_cases/Land_Use_Land_Cover/Visualization
+
+Next, configure the Apptainer bind mounts. This environment variable tells Apptainer how to map local directories and container images to paths inside the running container::
+
+   export APPTAINER_BIND="${IWRF_WORK_DIR}/data-lulc-input-obs.sif:/data/input/obs:image-src=/,${LOCAL_METPLUS_CONFIG_DIR}:/config,${IWRF_WORK_DIR}/data-lulc-input-wrf.sif:/data/input/wrf:image-src=/,${LOCAL_OUTPUT_DIR}:/data/output,${LOCAL_PLOT_SCRIPT_DIR}:/plot_scripts,${APPTAINER_TMPDIR}:${APPTAINER_TMPDIR}"
+
+This configuration provides the container with access to:
+
+* Observational data from the ``data-lulc-input-obs.sif`` container image at ``/data/input/obs``
+* WRF simulation data from the ``data-lulc-input-wrf.sif`` container image at ``/data/input/wrf``
+* METplus configuration files from the I-WRF repository at ``/config``
+* Visualization script files for generating plots at ``/plot_scripts``
+* Output directory for writing verification results at ``/data/output``
+* Temporary directory for Apptainer operations
+
+Execute METplus Verification
+----------------------------
+
+The LULC use case includes verification for two meteorological variables: accumulated precipitation and radar reflectivity. Each verification is run separately using its own METplus configuration file.
+
+Run the accumulated precipitation verification::
+
+   apptainer exec ${IWRF_WORK_DIR}/iwrf-metplus.sif /metplus/METplus/ush/run_metplus.py /config/GridStat_apcp_lulc.conf
+
+This process compares simulated precipitation accumulation against observational data and generates statistical metrics. Progress information is displayed while the verification is performed.
```

Comment:
```suggestion
This process compares simulated precipitation accumulation against observational data and generates statistical metrics. While this may take up to 20 minutes to run, progress information is displayed while the verification is performed.
```